rendermodules/rough_align/apply_rough_alignment_to_montages.py

```diff
@@ -141,28 +249,38 @@ def apply_rough_alignment(render,
                     keep_ref_tforms=True)
                 t.tforms = newt
             t.z = newz
-            #t.z = z
-            t.layout.sectionId = "%s.0"%str(int(newz))
-        
+            t.layout.sectionId = "%s.0" % str(int(newz))
+
+        if filter_montage_output_with_masks:
+            tf.M[0:2, 0:2] /= scale
```

Comment:
can you explain the logic behind this statement

Reply:
I want this function to use a lowres tilespec, without modification:
https://github.com/AllenInstitute/render-modules/blob/4b13871fcd0a4fe1f3753e78f0aa5106943e8051/rendermodules/rough_align/apply_rough_alignment_to_montages.py#L257

but here, the lowres tilespec is altered:
https://github.com/AllenInstitute/render-modules/blob/4b13871fcd0a4fe1f3753e78f0aa5106943e8051/rendermodules/rough_align/apply_rough_alignment_to_montages.py#L197-L198

So, I am undoing the scaling. I was trying not to change the existing code much.